homeworks/11/Domashno/src/main/java/com/example/domashno/DatabaseConnection.java

```diff
@@ -0,0 +1,27 @@
+package com.example.domashno;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+// This class can be used to initialize the database connection
+public class DatabaseConnection {

```

Comment:
As the comment states this class can be used for database connection, but it is never used. I suggest using it everywhere db connection is needed